src/aura/CRLP_FilterGroup/CRLP_FilterGroupHelper.js

```diff
@@ -31,9 +32,18 @@
                 cmp.set("v.isReadOnly", false);
             } else if (mode === "create") {
                 cmp.set("v.isReadOnly", false);
+                cmp.set("v.rollupItems",[]);
```

Comment:
also here, were rollupItems carrying over when new FGs were created? I'm wondering if this was an issue with clone too.

Reply:
great point. moved to a default instead.